### PR TITLE
New nav on about page

### PR DESCRIPTION
**Pull request by jeffrummel**
https://deploy-preview-426--poor-richard-spotlightpa.netlify.app/about/

### CODE REVIEW

File: layouts/partials/tw/footer.html [hunk 1]
```diff
@@ -29,7 +29,11 @@
       >
         <span
           class="
+<<<<<<< HEAD
```
Comment (carlmjohnson) [inline layouts/partials/tw/footer.html:32]: This is a merge conflict.

**Comment by carlmjohnson** [comment]
Screenshot 2023-06-07 at 1 20 53 PM

It feels like we are repeating the header a bunch. I think maybe we just ship this for now but revisit later to remove the middle black layer.

File: layouts/partials/tw/footer.html (hunk 1 above)
Comment (jeffrummel) [reply layouts/partials/tw/footer.html]: 😬 This is fixed

**Comment by jeffrummel** [comment]
> Screenshot 2023-06-07 at 1 20 53 PM
> 
> It feels like we are repeating the header a bunch. I think maybe we just ship this for now but revisit later to remove the middle black layer.

This is a quick fix I can deploy. 

Screenshot 2023-06-07 at 14-40-54 Spotlight PA Pennsylvania news and investigations

**Review by carlmjohnson — APPROVED** [review]

**Pull request merged**